docs/pages/modules/third-party-library.mdx

```diff
@@ -0,0 +1,335 @@
+import Charts
+
+struct Series: Record {
+  @Field
+  var color: UIColor = UIColor.black
+
+  @Field
+  var percentage: Double = 0
+}
+
+class ExpoRadialChartView: ExpoView {
+  let chartView = PieChartView()
+
+  required init(appContext: AppContext? = nil) {
+    super.init(appContext: appContext)
+    clipsToBounds = true
+    addSubview(chartView)
+  }
+
+  override func layoutSubviews() {
+    chartView.frame = bounds
+  }
+
+  func setChartData(data: [Series]) {
+    let set1 = PieChartDataSet(entries: data.map({ (series: Series) -> PieChartDataEntry in
+      return PieChartDataEntry(value: series.percentage)
+    }))
+    set1.colors = data.map({ (series: Series) -> UIColor in
+      return series.color
+    })
+    let chartData: PieChartData = [set1]
+    chartView.data = chartData
+  }
+}
+```
+
+We also need to use the [`Prop`](/modules/module-api/#prop) function to define the `data` prop and call the native `setChartData` function when the prop changes.
+
+```swift ios/ExpoRadialChartModule.swift
+import ExpoModulesCore
+
+public class ExpoRadialChartModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoRadialChart")
+
+    View(ExpoRadialChartView.self) {
+      Prop("data") { (view: ExpoRadialChartView, prop: [Series]) in
+        view.setChartData(data: prop)
+      }
+    }
+  }
+}
+```
+
+</Step>
+
+<Step label="7">
+
+## Write an example application to use our module
+
+Now we can update the app inside the `example` directory to to test our module. We'll use the `ExpoRadialChartView` component to render a pie chart with three slices.
+
+```tsx example/App.tsx
+import { ExpoRadialChartView } from 'expo-radial-chart';
+import { StyleSheet } from 'react-native';
+
+export default function App() {
+  return (
+    <ExpoRadialChartView
+      style={styles.container}
+      data={[
+        {
+          color: '#ff0000',
+          percentage: 0.5,
+        },
+        {
+          color: '#00ff00',
+          percentage: 0.2,
+        },
+        {
+          color: '#0000ff',
+          percentage: 0.3,
+        },
+      ]}
+    />
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+});
+```
+
+> **Tip**: If you created the module inside an existing application, make sure to import it directly from your `modules` directory. It's easiest to use a relative import. `import { ExpoRadialChartView } from '../modules/expo-radial-chart';`
+
+</Step>
+
+<Step label="8">
+## Rebuild and launch your application
+
+Rerun the build commands from step 2 and make sure your application builds successfully on both platforms.
+
+You should see a pie chart with three slices:
+
+![A PieChart module on iOS and Android](/static/images/modules/third-party-library/result.webp)
+
+Congratulations, you have created your first simple wrapper around two separate native libraries using Expo Modules! Learn more about the API in the [Expo Module API reference](/modules/module-api/).
```

Comment:
```suggestion
</Step>

## Next steps

Congratulations! You've created your first simple wrapper around two separate third-party native libraries using Expo Modules!
```

Let's use two Box link component here to refer the below links:
- Learn more about the API in the [Expo Module API reference](/modules/module-api/).
- creating a native module" tutorial](/modules/native-module-tutorial/)

You can use the same description in those components as did to describe them here.